src/tests/ftest/pool/evict.yaml

```diff
@@ -3,16 +3,13 @@ hosts:
 
 server_config:
   name: daos_server
+  system_ram_reserved: 1
```

Comment:
Why is this 1 and not 2, like the other VM test yamls?

Reply:
as above, done